src/main/java/edu/greenblitz/robotName/commands/shooter/ShootByShootingPower.java

```diff
@@ -0,0 +1,30 @@
+package edu.greenblitz.robotName.commands.shooter;
+
+import edu.greenblitz.robotName.commands.shooter.flyWheel.FlyWheelCommand;
+import edu.greenblitz.robotName.commands.shooter.flyWheel.StopFlyWheel;
+import edu.greenblitz.robotName.commands.shooter.funnel.RunFunnel;
+import edu.greenblitz.robotName.subsystems.shooter.FlyWheel.FlyWheelConstants;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+
+public class ShootByShootingPower extends FlyWheelCommand {
+
+    @Override
+    public void execute() {
+        flyWheel.setPower(FlyWheelConstants.SHOOTING_POWER);
+    }
+
+    @Override
+    public boolean isFinished() {
+        return flyWheel.isAtVelocity(FlyWheelConstants.SHOOTING_POWER);
+    }
+
+    @Override
+    public void end(boolean interrupted) {
+        new SequentialCommandGroup(
+                new RunFunnel(),
+                new WaitCommand(FlyWheelConstants.DELAY_SECONDS_TILL_EXIT),
+                new StopFlyWheel()
+        ).schedule();
```

Comment:
no.
split this into two commands or find a better solution, don't schedule commands inside other commands